config/initializers/locations.rb

```diff
@@ -1 +1 @@
-Locations.path = Rails.root.join('lib', 'locations', 'locations.json')
+Locations.path = 'http://0.0.0.0:8080/locations.json'
```

Comment:
I think the quickest thing is to name it SUPER explicitly, something like `Locataions.geojson_path_or_url`, and move on with our lives